src/ngraph/runtime/cpu/cpu_call_frame.cpp

```diff
@@ -71,6 +74,47 @@ void runtime::cpu::CPU_CallFrame::call(
         m_external_function->get_executor()(ctx, inputs, outputs);
     }
 
+    if ((std::getenv("NGRAPH_DEX_DEBUG") != nullptr) && (std::getenv("NGRAPH_DEX") != nullptr))
```

Comment:
maybe we could refactor this into a standalone function? It's a fairly considerable chunk of tracing logic